src/pymmcore_widgets/_mda/_stack_viewer.py

```diff
@@ -200,7 +205,9 @@ def on_display_timer(self, value: int, name: str) -> None:
             self.display_index[slider.name] = slider.value()
         if old_index == self.display_index:
             return
-        for c in range(self.sequence.sizes["c"]):
+        if (sequence := self.sequence) is None:
```

Comment:
since we defined it as `MDASequence | None`, we either need to use a `cast` here, or check that it's not None